openqa-label-known-issues

```diff
@@ -156,11 +157,24 @@ investigate_issue() {
     # subject line
     elif label_on_issues_without_tickets "$id"; then return
     else
-        handle_unknown "$i" "$out"
+        handle_unknown "$i" "$out" "$reason"
     fi
     rm "$out"
 }
 
+print_summary() {
+    local to_review_count=${#to_review[@]}
+    if [[ $to_review_count -eq 0 ]]; then
+        echoerr -e "\n\e[1m\e[32mNo unknown incompletes to be reviewed.\e[0m"
```

Comment:
I would prefer to use the common *nix style to not output anything if there was no problem.

Reply:
I prefer having an output similar to what most test frameworks provide (a green line which states everything's fine).

Reply:
How about we use something well-defined like TAP?

Reply:
I prefer to have *something* better as soon as possible and would postpone and extra wishes for now. Also note that this is a shell script so we can't directly use a fancy library to do all kinds of formatting.

Reply:
TAP is a specification, not a library. My point was that neither "nix style" nor "most test frameworks" is very meaningful and we are used to TAP and know what format to use as we extend it.

```
ok 1 - No unknown incompletes to be reviewed
```

```
not ok 1 - unknown incompletes to be reviewed
#  Job 12345
# Looks like you failed 1 test of 1
```

Reply:
how about adding an output only on request, e.g. based on command line parameter. Think about the "happy path" which is that either no incompletes are there or all tickets could be automatically labeled. In this case I like that there is no output so I understand there is no action to be done. An the script is designed so that we could rerun it, see there is (still) no output and we know there is not more work needed. If I would need to read the text which says "No action needs to be done" then this would be more distracting.